docs/advanced_usage/how_to_auxiliary.rst

```diff
@@ -4,33 +4,23 @@ How to create an auxiliary
 ==========================
 
 This tutorial aims to explain the working principle of an Auxiliary by
-providing information on the different Auxiliary interfaces, their purpose
+providing information on available Auxiliary interface, their purpose
```

Comment:
Please, also update the examples folder. I am pretty sure there are some examples related to old auxes interfaces(see templates folder ...).